src/PdfParser/Type/PdfDictionary.php

```diff
@@ -107,7 +107,7 @@ public static function create(array $entries = [])
      * @return PdfNull|PdfType
      * @throws PdfTypeException
      */
-    public static function get($dictionary, $key, PdfType $default = null)
+    public static function get($dictionary, $key, $default = null)
```

Comment:
Should we also change the return logic to explicity check for `PdfType` then to avoid a possible return type change?

    return (!$default instanceof PdfType)
        ? new PdfNull()
        : $default;

Reply:
I added a check at the beginning of the method, which is closer to what PHP's argument type checking would do.